src/randoop/SerializableExpectedExceptionChecker.java

```diff
@@ -17,9 +23,9 @@ public SerializableExpectedExceptionChecker(Class<? extends Throwable> exception
     this.statementIdx = statementIdx;
   }
 
-  private Object readResolve() throws ObjectStreamException {
-    @SuppressWarnings("unchecked")
-    Class<? extends Throwable> c = (Class<? extends Throwable>) Reflection.classForName(exceptionClass);
+  @SuppressWarnings("unchecked")
```

Comment:
What is the reason for changing the `@SuppressWarnings`?  Before it applied to only one variable declaration; now it applies to the whole body of the method.  I think it would be better to keep its scope as small as possible, for clarity as to what it is doing and to avoid suppressing unintended warnings.